tests/ml_commons/test_ml_commons_client.py

```diff
@@ -147,6 +150,7 @@ def test_integration_model_train_upload_full_cycle():
 
                 try:
                     ml_client.unload_model(model_id)
+                    time.sleep(30)
```

Comment:
Sleep is flaky as sometimes it may take longer time than hard coded timeout. Use waiting to check task status every second? https://pypi.org/project/waiting/